src/Fable.AST/Fable.fs

```diff
@@ -266,12 +267,18 @@ type KeyKind =
 type GetKind =
     | ByKey of KeyKind
     | TupleIndex of int
+    | FieldGet of Field * index: int
```

Comment:
Should we add `Field` to the AST elements or maybe just the name so we can retrieve the field from the entity reference when needed as in here? https://github.com/fable-compiler/Fable/pull/2447#discussion_r640307157

Reply:
Same for `SetKind` and `KeyKind` (which already contain a direct reference to `Field` but maybe we can remove it).

Reply:
@alfonsogarciacaro I think adding Field is fine, but whatever yo prefer.